Dockerfile

```diff
@@ -9,41 +9,47 @@ ENV LANG="en_US.UTF-8" \
     SINUS_DIR="/sinusbot" \
     YTDL_BIN="/usr/local/bin/youtube-dl" \
     YTDL_VERSION="latest" \
-    TS3_VERSION="3.0.18.2" \
-    TS3_DL_ADDRESS="http://teamspeak.gameserver.gamed.de/ts3/releases/" \
-    SINUSBOT_DL_URL="https://www.sinusbot.com/dl/sinusbot-beta.tar.bz2"
+    TS3_VERSION="3.1.8" \
+    TS3_DL_ADDRESS="http://dl.4players.de/ts/releases/" \
```

Comment:
Is there a reason to change the download to `4players.de`?

Reply:
No, just opened http://teamspeak.gameserver.gamed.de/ts3/releases/ in Chrome which gives me a 404 Error. Thought it may be better to change to someone where you can see whats behind.